mkosi/__init__.py

```diff
@@ -5368,6 +5383,14 @@ def create_parser() -> ArgumentParserMkosi:
         default=0,
         help="Wait up to SECONDS seconds for the SSH connection to be available when using 'mkosi ssh'",
     )
+    group.add_argument(
+        "--ssh-agent",
+        type=parse_ssh_agent,
+        default=False,
+        metavar="PATH",
+        help="If True, 'mkosi ssh' will use $SSH_AUTH_SOCK (hint: use sudo -E) to talk to the ssh agent socket "
+        "to fetch and use the private key. Can also be used to manually pass the path to the ssh agent socket.",
```

Comment:
So this description is misleading. I primarily uses the agent to fetch the public key list…